demo/demo-stack.vue

```diff
@@ -9,6 +9,9 @@
 			<h1>CompA</h1>
 			<button @click="state.bottomSheet = !state.bottomSheet">Toggle</button>
 			<button @click="addStack">Add</button>
+			<gl-drag :component="DragTest" componentName="DragTest" :data="{}">
```

Comment:
Take care here that `<gl-...` components are met to be put beside other `<gl-...` components. There is no `<gl-...` in a `<div>` and there is a `<div>` only in a `<gl-component>`